svg-tag-mode.el

```diff
@@ -158,23 +158,25 @@ This should be zero for most fonts but some fonts may need this."
 
 (defun svg-tag-mode-on ()
   (add-to-list 'font-lock-extra-managed-props 'display)
-  (if active-svg-tags
-      (font-lock-remove-keywords nil active-svg-tags))
-  (if svg-tags
-      (font-lock-add-keywords nil svg-tags))
-  (setq active-svg-tags (copy-sequence svg-tags))
+  (when svg-tags--active-tags
+    (font-lock-remove-keywords nil svg-tags--active-tags))
+  (when svg-tags
```

Comment:
What's the difference with ìf`?

Reply:
No internal differences here but `when` is more idiomatic when there's only one branch